webpack.config.js

```diff
@@ -44,6 +44,7 @@ const CSS_CONFIG = {
 };
 
 const config = (module.exports = {
+  mode: devMode ? "development" : "production",
```

Comment:
# Add `mode` to the config

```
[frontend] ERROR in ../../../node_modules/leaflet/dist/images/layers-2x.png
[frontend] Module build failed (from ../../../node_modules/file-loader/index.js):
[frontend] TypeError: Cannot read property 'fileLoader' of undefined
[frontend]     at Object.module.exports (/Users/offbynone/code/work/metabase/node_modules/file-loader/index.js:14:28)
[frontend]  @ ../../../node_modules/css-loader/index.js??ruleSet[1].rules[4].use[1]!../../../node_modules/postcss-loader/lib/index.js!../../../node_modules/leaflet/dist/leaflet.css 7:8841-8874
[frontend]  @ ../../../node_modules/leaflet/dist/leaflet.css 4:14-128 18:2-22:4 19:20-134
[frontend]  @ ./visualizations/components/LeafletChoropleth.jsx 46:0-35
[frontend]  @ ./visualizations/components/ChoroplethMap.jsx 92:25-55
[frontend]  @ ./visualizations/visualizations/Map.jsx 85:21-59
[frontend]  @ ./visualizations/register.js 79:11-42
[frontend]  @ ./app.js 28:16-59
[frontend]  @ ./app-public.js 3:11-27
```
We're using `file-loader` version 0.11.1. The latest one is 6.2.0! Let's update it in the next step.